carLot.js

```diff
@@ -0,0 +1,23 @@
+var CarLot = (function (load) {
+	var inventory = [];
+
+	load.loadInventory= function (callback) {
```

Comment:
```callback``` was included here in the boilerplate to indicate that you're to pass in a callback function, your code would be more readable if you replace this argument name with something more evocative of what the callback function does, or with the name of the callback function itself.